README.md

```diff
@@ -1 +1,238 @@
-# mod-eventbus3-bridge-client
+# Vert.x3 eventbus bridge client for Vert.x2
+
+This project will allow bidirectional communication between a vert.x2 and a vert.x3 application.
+
+## Installation
+
+Add the following dependency to your project:
+
+### maven
+
+```xml
+<dependency>
+  <groupId>io.vertx</groupId>
+  <artifactId>mod-eventbus3-bridge-client</artifactId>
+  <version>1.0.0</version>
```

Comment:
I don't know what the best is. While the protocol stays untouched on v3 this bridge should just work and does not need releases whenever a new 3 release is done...